digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/ProjectService.scala

```diff
@@ -767,5 +768,24 @@ class ProjectService(roadAddressService: RoadAddressService, roadLinkService: Ro
 
   }
 
+  def setProjectEly(currentProjectId:Long, newEly: Long): String = {
+    val currentProjectEly = getProjectEly(currentProjectId)
+    if (currentProjectEly == -1) {
+      ProjectDAO.updateProjectEly(currentProjectId, newEly)
+      ""
+    } else if (currentProjectEly == newEly){
+      logger.info("ProjectId: " + currentProjectId + " Ely is \"" + currentProjectEly + "\" no need to update")
+      ""
+    } else {
+      logger.error("Cannot update project to another ELY, current road can't be added")
+      //TODO: translate or find a suitable message in Finnish
```

Comment:
I just need a small clarification, Is this phrase:
"Virhe: Projektia ei voi päivittää toiselle ELY: lle, nykyistä tieä ei voi lisätä"
Ok or should it be switched to another?